docs/csharp/programming-guide/classes-and-structs/index.md

```diff
@@ -52,9 +51,9 @@ Classes and structs are two of the basic constructs of the common type system in
  For more information, see [Classes](../../../csharp/programming-guide/classes-and-structs/classes.md), [Objects](../../../csharp/programming-guide/classes-and-structs/objects.md), and [Structs](../../../csharp/programming-guide/classes-and-structs/structs.md).  
   
 ## Example  
- In the following example, `MyCustomClass` is defined with three members at the top level of the `ProgrammingGuide` namespace. An instance (object) of `MyCustomClass` is created in the `Main` method in the `Program` class, and the object’s methods and properties are accessed by using dot notation.  
+ In the following example, `customClass` in the `ProgrammingGuide` namespace has three members: an instance constructor, a property named `Number`, and a method named `Multiply`. The `Msin` method in the `Program` class creates an instance (object) of `myCustomClass`, and the object’s method and property are accessed by using dot notation.  
```

Comment:
also, a typo: Msin should be Main and myCustomClass should be just CustomClass?

in addition to that: dot notation -> the dot notation?